docs/docs/20-quickstart.mdx

```diff
@@ -25,7 +25,14 @@ This guide presents a basic introduction to Kargo. Together, we will:
     * kind: v0.17.0
     * k3d: v5.4.9
 * [Helm](https://helm.sh/docs/): These instructions were tested with v3.11.2.
-* The `kargo` CLI: Download it from [the releases page](https://github.com/akuity/kargo/releases/latest).
+* The `kargo` CLI: To install the kargo CLI, follow these steps.
+
+    1. Visit [the releases page](https://github.com/akuity/kargo/releases/latest) of the Kargo GitHub repository
+    2. Download the appropriate release package for your operating system (e.g., `kargo-linux-amd64` for Linux).
+    3. Extract the contents of the downloaded package to a directory of your choice.
+    4. Add the directory containing the `kargo` executable to your system's `PATH` environment variable.
```

Comment:
This is great for now! However, we should include the commands to complete these steps. That would probably be better served in its own "Installing the CLI" section of the docs.